lib/create-response-handler.ts

```diff
@@ -1,46 +1,51 @@
 /**
+ * The contents of this file are only relevant if your service uses the
```

Comment:
How about
'Use this file only if your service is using the legacy setup'
?

Reply:
The thing is, they don't exactly use the file, they use one of the functions declared in the file.  I prefer the current wording.